lib/Transport.js

```diff
@@ -56,6 +62,11 @@ Transport.prototype._action = function(method, url, data, callback) {
 
       return callback(realErrors[0]);
     }
+
+    if (!payload.body) {
+      payload.body = JSON.parse(payload.text);
+    }
```

Comment:
Again, another bug/feature in `superagent` whereby it won't always correctly JSON parse an `application/json` payload.